### DISCUSSION TITLE
Updating dependencies

### DISCUSSION THREAD

**Discussion by mdiepart** [lora-aprs/LoRa_APRS_iGate#249] [General]
Hi @peterus,

I could not help but notify that you added a bot to help updating dependencies.
There are major updates for some of them which will require you to update some of your libraries on platformio.
I already updated most of them on my fork (branch web_server ) (espressif32 is only updated to version 5.3.0, version 5.4.0 will break something with FTP library).

If you want to discuss that here or in an issue, let's do that !

73,
Morgan

**Comment by peterus** [comment]
Hi Morgan!

😂
You are right! I am currently working on a script for platformio which should do some basic things like dependabot.
The main purpose is to stay on the last releases with all the libraries and platforms.

As i am very far behind with external releases in this repo, thats the reason why i used it to test my script.

I did made some good progress yesterday, but still want to add some small things (like adding the release texts to the push request etc.).

The script (and some examples) can be found here: https://github.com/peterus/platformio_dependabot
Currently the repo is empty, but will fill it in the next days. There is also a plan to use it in github actions.

As you have already seen: the first stepp will be to fix the ftp library 😊

**Comment by peterus** [comment]
Hi Morgen!

All dependencies are now updated.
The Dependabot script is working good for PlatformIO :)

73 Peter (OE5BPA)
